test/packer_basic_example_test.go

```diff
@@ -15,6 +17,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// Occasionally, a Packer build may fail due to intermittent issues (e.g., brief network outage or EC2 issue). We try
+// to make our tests resilient to that by specifying those known common errors here and telling our builds to retry if
+// they hit those errors.
+var DefaultRetryablePackerErrors = map[*regexp.Regexp]string{
+	regexp.MustCompile("Script disconnected unexpectedly"): "Occasionally, Packer seems to lose connectivity to AWS, perhaps due to a brief network outage",
```

Comment:
This is an intermittent Packer error I hit recently that motivated this PR. However, I've hit many others before, but just can't find the CI builds to copy down the error messages.

Can anyone else find error messages for intermittent Packer build failures so we can include retries for them here?

Reply:
Yeah here are two from my side:

1. Build: https://circleci.com/gh/gruntwork-io/terratest/1610
Failed Test Log: https://1610-53174289-gh.circle-artifacts.com/0/tmp/logs/TestPackerMultipleConcurrentAmis.log

2. Build: https://circleci.com/gh/gruntwork-io/terratest/1606
Failed Test Log: https://1606-53174289-gh.circle-artifacts.com/0/tmp/logs/TestPackerGCPBasicExample.log

The second one should be resolved by one of the patches I pushed, but there seems to be a bug with it.

Reply:
The first error, I believe, is `Script disconnected unexpectedly`, which I already have in `DefaultRetryablePackerErrors`. The second one happens before the Packer build, so this PR won't be able to help there.

Reply:
Also, I just updated the retry logic for the second error to use the existing `retry.DoWithRetry` function: https://github.com/gruntwork-io/terratest/pull/311/commits/096157c5b62cd7d2c9d9c495c3bc9c225c70a11b